### DISCUSSION TITLE
Ease widget configuration for fields

### DISCUSSION THREAD

**Discussion by ponceta** [teksi/wastewater#447] [Ideas]
Friday morning idea / Ease widget configuration for fields

Copy paste widget / default values / constraints
- [ ] https://github.com/qgis/QGIS/issues/58971

Show widget type
- [ ] https://github.com/qgis/QGIS/issues/58973

Show warning on obsolete or broken relation widget
- [x] https://github.com/qgis/QGIS/issues/59001

@urskaufmann a good idea?

**Comment by urskaufmann** [comment]
yes - for all of these organisation value relations, also in case of changes e.g. filter

another idea: show the widget type of every field in the layer properties/attribut form. Now I have to click trough every field to control, if the config is done.
![Annotation 2024-10-04 104018](https://github.com/user-attachments/assets/3ac49ee2-07e2-4568-812e-6b3c39cfab11)

**Reply by ponceta** [reply]
Bringing your excellent proposal as a second QGIS feature request :
https://github.com/qgis/QGIS/issues/58973

**Comment by opengis-ch** [comment]
@ponceta is there a priority between these tickets or they have the same?

**Reply by ponceta** [reply]
Not from our side, but as I see it we have :
- A full QGIS Feature
- A "simple" QGIS UI enhancement
- A bug that require some "warnings" when occuring

I would then go from the simplest to the fullest meaning :

1. https://github.com/qgis/QGIS/issues/59001
2. https://github.com/qgis/QGIS/issues/58973
3. https://github.com/qgis/QGIS/issues/58971

item 3. Will probably require some QGIS-CH cofinancement.

**Comment by 3nids** [comment]
for the record:
* https://github.com/qgis/QGIS/issues/59001 is solved
* https://github.com/qgis/QGIS/issues/58973 we will publish a QEP in the following days
* https://github.com/qgis/QGIS/issues/58971 @gacarrillor will finalize the implementation (hopefully still in this release cycle - before Jan. 17th)

**Comment by gacarrillor** [comment]
@ponceta Note the _Show widget type_ issue has been addressed and is already available in QGIS master. See https://github.com/qgis/QGIS/issues/58973#issuecomment-3462520366 for details

**Reply by ponceta** [reply]
I'll give it a try tokay! Thank you for the great work!
